Makefile

```diff
@@ -0,0 +1,14 @@
+## DO NOT EDIT!
+# This file was provisioned by Terraform
+# File origin: https://github.com/Arrow-air/tf-github/tree/main/src/templates/all/Makefile
+
+DOCKER_NAME := arrow-${name}
+
+help: .help-base .help-cspell .help-markdown .help-editorconfig
```

Comment:
toml, python, docker helps and includes?

Reply:
I don't think they'll be needed for the tf-github repo, although, maybe toml would be helpful to check here too